index.html

```diff
@@ -1,12 +1,22 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
+<link rel="stylesheet" href="main.css">
   <head>
     <meta charset="utf-8">
-    <title></title>
-  <!-- need to add link to css -->
+    <meta name="keywords" content="Dashboard">
+    <title>Dashboard</title>
+    <h1>
+      <!--Insert navigation bar here-->
+    </h1>
+    <h2>
+      Dashboard
+    </h2>
   </head>
```

Comment:
all of the contents of the website are supposed to be in the <body> tags, not the <head> tags. 

Think of <head> as <headquarters> where you do thinks like <link> your stylesheet, set your <title>, make tags using <meta author="">

Then <body> is the visuals you're displaying to the user.